src/grt/src/fastroute/src/utility.cpp

```diff
@@ -2022,7 +1955,7 @@ void FastRouteCore::copyBR(void)
         v_edges_[i][j].usage = 0;
       }
     }
-    for (netID = 0; netID < netCount(); netID++) {
+    for (int netID = 0; netID < netCount(); netID++) {
```

Comment:
Similar question here. Why use `netCount()` if you have the vector with the net IDs?